networkx/algorithms/centrality/group.py

```diff
@@ -93,40 +100,134 @@ def group_betweenness_centrality(G, C, normalized=True, weight=None):
        Group Centrality Maximization via Network Design.
        SIAM International Conference on Data Mining, SDM 2018, 126–134.
        https://sites.cs.ucsb.edu/~arlei/pubs/sdm18.pdf
+    .. [4] Rami Puzis, Yuval Elovici, and Shlomi Dolev.
+       "Fast algorithm for successive computation of group betweenness centrality."
+        https://journals.aps.org/pre/pdf/10.1103/PhysRevE.76.056709?casa_token=YvH6xTIc6AUAAAAA%3A90BkxMRiQyOg9y7PEU_lrcf1FXPF_sqTCI_Y_2lD0QodSeyXAQtz-s0_e1Z0CtkZ1bWdVCkK9S5fXw
```

Comment:
```suggestion
        https://journals.aps.org/pre/pdf/10.1103/PhysRevE.76.056709
```
The `casa_token` part of the URL is not needed.

Reply:
changed